.buildkite/commands/build-for-testing.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+echo "--- :rubygems: Setting up Gems"
+restore_cache "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
+gem install bundler
+bundle install
+save_cache vendor/bundle "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
+
+echo "--- :cocoapods: Setting up Pods"
+
+# Caching the specs repos and global pod cache can dramatically improve Pod times
+restore_cache "$BUILDKITE_PIPELINE_SLUG-specs-repos"
+restore_cache "$BUILDKITE_PIPELINE_SLUG-global-pod-cache"
+
+restore_cache "$(hash_file Podfile.lock)"
```

Comment:
Similar to my previous comment, it might be worth extracting the key.

Reply:
I guess same goes for `"$BUILDKITE_PIPELINE_SLUG-specs-repo"` & `"$BUILDKITE_PIPELINE_SLUG-global-pod-cache"`. I see that [@mokagio left a similar comment.](https://github.com/wordpress-mobile/WordPress-iOS/pull/16787/files#r676240069)